test/test_structured_2d.jl

```diff
@@ -22,14 +22,13 @@ isdir(outdir) && rm(outdir, recursive=true)
   @trixi_testset "elixir_advection_coupled.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_coupled.jl"),    
       l2   = [7.816742843181738e-6, 7.816742843196112e-6],
-      linf = [6.314906965543265e-5, 6.314906965410039e-5],
-      coverage_override = (maxiters=10^5,))
+      linf = [6.314906965543265e-5, 6.314906965410039e-5])
+  end
 
-    @testset "analysis_callback(sol) for AnalysisCallbackCoupled" begin
-      errors = analysis_callback(sol)
-      @test errors.l2   ≈ [7.816742843181738e-6, 7.816742843196112e-6] rtol=1.0e-4
-      @test errors.linf ≈ [6.314906965543265e-5, 6.314906965410039e-5] rtol=1.0e-4
-    end
```

Comment:
Did you remove this on purpose?

Reply:
Yes, it caused problems in the tests.

Reply:
What kind of problems? And do you still check what this test intended to check?

Reply:
Maybe not relevant anymore. Since coupling is going to happen with converters by default `elixir_advection_coupled.jl` is somewhat outdated. I have replaced it with the old `elixir_advection_coupled_converter.jl`.